pnpm-lock.yaml

```diff
@@ -16,14 +16,14 @@ importers:
     dependencies:
       '@logto/cli': link:packages/cli
     devDependencies:
-      '@changesets/cli': 2.25.0
-      '@commitlint/cli': 17.0.0
-      '@commitlint/config-conventional': 17.0.0
-      '@commitlint/types': 17.0.0
+      '@changesets/cli': 2.26.0
```

Comment:
i think this size of change is unexpected. you can check out your lockfile and run `pnpm i` again

Reply:
Updated.